src/sentry/static/sentry/app/components/organizations/timeRangeSelector/dateRange/selectorItem.jsx

```diff
@@ -33,12 +37,13 @@ const StyledSelectorItem = styled(SelectorItem)`
   padding: ${space(1)};
   align-items: center;
   flex: 1;
-  background-color: ${p => (p.selected ? p.theme.offWhite : 'transpatent')};
+  background-color: ${p => (p.selected ? p.theme.offWhite : 'transparent')};
   font-weight: ${p => (p.selected ? 'bold' : 'normal')};
   border-bottom: 1px solid ${p => (p.last ? 'transparent' : p.theme.borderLight)};
+  color: ${p => p.disabled && p.theme.disabled};
 
   &:hover {
-    background: ${p => p.theme.offWhite};
+    background: ${p => !p.disabled && p.theme.offWhite};
```

Comment:
```suggestion
    ${p => !p.disabled && `background: ${p.theme.offWhite};`}
```

Right now when the input is disabled you'll emit invalid CSS.

Reply:
Hm I thought the attribute just gets dropped

Reply:
It does, the browser ignores the property as its bogus.

Reply:
I actually thought emotion dropped the property if it was null.